LiftLog.Lib/Models/SessionModels.cs

```diff
@@ -102,14 +102,14 @@ public RecordedExercise? NextExercise
 
     public TimeSpan SessionLength =>
         RecordedExercises
-            .Select(x => x.LastRecordedSet?.Set?.CompletionTime)
+            .Select(x => x.LastRecordedSet?.Set?.CompletionDate.ToDateTime(x.LastRecordedSet.Set.CompletionTime))
```

Comment:
With the previous comment:
```suggestion
            .Select(x => x.LastRecordedSet?.Set?.CompletionDateTime
```